app.rb

```diff
@@ -0,0 +1,20 @@
+require 'sinatra/base'
+
+class ChitterApp < Sinatra::Base
+  
+  get '/' do
+    "Welcome to Chitter"
+  end
+
+  get '/peep' do
+    erb :index
+  end
+
+  get '/peep/new' do
+    @peep = Peep.all
+    erb :peep
+  end
```

Comment:
Clean controller. Could maybe have it even cleaner by having the root show your index view!